content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,381 @@
+
+Rewards are paid for reporting a slashable _fault_ at epoch end. The _offending_ validator's share of the epoch's staking rewards is forefeited and paid to the  _reporting_ validator `treasury` account for distribution along with the staking rewards at epoch end.
+
+### Accountability prerequisites 
+
+To participate in AFD a [validator](/glossary/#validator) must be a [consensus committee](/glossary/#consensus-committee) member.
+
+## Accountability and Fault Detection protocol
+
+AFD roles, core concepts, and the lifecycle of accountability event processing from accusation to slashing.
+
+### Validator roles
+
+As consensus committee members, validators play the following roles in AFD processing:
+
+- _reporting_: as the validator reporting a suspected rule infraction and submitting new _accusation_ proofs on-chain
+- _offending_: as the validator committing a suspected rule infraction and submitting new _innocence_ proofs on-chain
+- _committee member_: as a validator in the consensus committee executing Autonity's consensus protocol and for AFD handling and processing accountability events, maintaining system state for accountability events, and computing and applying slashing penalties.
+
+Validators play the following economic roles in AFD processing:
+
+- _loss_: loss of stake, validator reputation, staking reward revenue as the _offending_ validator of a slashed _fault_
+- _beneficiary_: gain of slashing reward as the _reporting_ validator of a slashed _fault_.
+
+Autonity community play the following economic role in AFD processing:
+
+- _beneficiary_: gain of slashed stake token for community funds.
+
+### Protocol primitives
+
+Essential primitives of AFD are: accusation, innocence, and fault proofs; slashing and severity; jailing.
+
+#### Accusations
+
+An _accusation_ is a claim that a consensus committee member has failed to participate correctly in consensus. I.e. it has failed to adhere to a rule governing validator participation in consensus and so is guilty of a rule infraction. Protocol only allows a validator to be under _accusation_ once at a time. New _accusations_ are made by submitting _proof_ of an accusable rule infraction on-chain as an event of type `NewAccusation`.
+
+Accusations do not automatically cause slashing and an _innocence proof window_ measured in blocks gives the accused _offending_ validator a window to detect an _accusation_ and prove _innocence_ by submitting an _innocence_ proof on-chain. The expiry of this window creates a _deadline_ before which a new `Accusation` proof cannot be submitted.
+
+There are protocol constraints on when an _accusation_ can be made. A _reporting_ validator can only submit an _accusation_ _proof_ if:
+
+- the _offending_ validator:
+  - does not already have a _fault_ in the epoch in which the _new accusation_ is being made for an offence with a higher _severity_
```

Comment:
It's not really the epoch where the accusation is made, but the epoch in which the "suspicious behaviour" happened. e.g.

If you someone accuses you for epoch x but they report the accusation in epoch y, what counts is epoch x

Reply:
Yes. I've clarified by changing "in the epoch in" --> "in the epoch for". Commit: f0abfb26c62401bfb3d3de49897a639a5944840c.

Reply:
Can this be resolved after this edit to clarify @lorenzo-dev1 , pls?